src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -208,4 +208,9 @@
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="System\Text\Json\Node\JsonBoolean.cs" />
+    <Compile Include="System\Text\Json\Node\JsonNode.cs" />
+    <Compile Include="System\Text\Json\Node\JsonString.cs" />
+  </ItemGroup>
```

Comment:
Nit: Why have a separate ItemGroup?

Reply:
Since there are still some changes being made around JSON for release/3.0, we had decided to keep the itemgroup for these new files separate to keep merge conflicts at a minimum when cherry-picking fixes from master to release. We'd merge the Itemgroup back once 3.0 is done.
https://github.com/dotnet/corefx/pull/39917#discussion_r310851213
https://github.com/dotnet/corefx/pull/39917#discussion_r310334520